substrate/client/rpc-spec-v2/src/archive/api.rs

```diff
@@ -104,4 +107,21 @@ pub trait ArchiveApi<Hash> {
 		items: Vec<PaginatedStorageQuery<String>>,
 		child_trie: Option<String>,
 	) -> RpcResult<ArchiveStorageResult>;
+
+	/// Returns the storage difference between two blocks.
+	///
+	/// # Unstable
+	///
+	/// This method is unstable and subject to change in the future.
```

Comment:
```suggestion
	/// This method is unstable and can change in minor or patch releases.
```

Or something like that maybe? :)